### PR TITLE
chore: update version constraints

### PR DESCRIPTION
**Pull request by iamar7**
### Description

https://github.ibm.com/GoldenEye/issues/issues/4915

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
please update `modules/fscloud/version.tf` too

**Review by ocofaigh — DISMISSED** [review]

**Comment by ocofaigh** [comment]
@iamar7 can you resolve conflicts please?

File: version.tf [hunk 1]
```diff
@@ -6,5 +6,10 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = ">= 1.56.1, < 2.0.0"
     }
+    # tflint-ignore: terraform_unused_required_providers
+    time = {
+      source  = "hashicorp/time"
+      version = ">= 0.9.1, < 1.0.0"
+    }
```
Comment (ocofaigh) [inline version.tf:13]: why add this? It was just removed in a recent PR

Comment (iamar7) [reply version.tf:13]: Removed it

Comment (ocofaigh) [reply version.tf:13]: @iamar7 I still see it?

**Pull request merged**